openapi/src/parts/resource.yaml

```diff
@@ -30,6 +30,31 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/{resourceId}:
```

Comment:
why is the existing get method not good enough for your usage?

Reply:
The [axon server getFile endpoint](https://docs.google.com/document/d/1_6ruIYYzTnbfrqfjEquRsWUx1ZsNJSzCALa9iMZ6au0/edit?resourcekey=0-L8V4uz_kouNZoUhvjO1R_A#heading=h.mo0z046y10jz) accepts general resource id. We need this endpoint to determine the resourceType and validate whether it is a file containing resource. Without this endpoint, we would unecessarily get all resources in the workspace with enumerateResources then filter for the desired resource.

Reply:
@yuhuyoyo I don't think there is an existing method that does this. Is there and I missed it?

Reply:
no. I meant we have endpoint per resources. i remember it was an intentional decision to not use the union attributes and implement endpoint per resources.

With this endpoint, why do we need GET per resource type at all?

i get why this is useful for the axon use case.

Reply:
Good point. When we do the "v2" interface design, maybe we skip the per-resource GET in favor of this style!?